common/plugins/ipam.go

```diff
@@ -112,7 +111,7 @@ func IPv4Routes(addr *models.NodeAddressing, linkMTU int) ([]Route, error) {
 		{
 			Prefix:  node.IPv4DefaultRoute,
 			Nexthop: &ip,
-			MTU:     linkMTU - mtu.TunnelOverhead,
+			MTU:     linkMTU,
```

Comment:
There's no tunnel overhead anymore?

Reply:
It's abstracted in the mtu package:
```
// GetRouteMTU returns the MTU to be used on the network. When running in
// tunneling mode, this will have tunnel overhead accounted for.
func GetRouteMTU() int {
        return TunnelMTU
}
```